### DISCUSSION TITLE
Change Kitten diff colour-scheme

### DISCUSSION THREAD

**Discussion by anurag3301** [kovidgoyal/kitty#3765] [Q&A]
Hey I just started using Kitty, I have set my colour scheme using this [GitHub Repo](https://github.com.cnpmjs.org/dexpota/kitty-themes)
My kitty looks like this
[![2021-06-25-221739-1364x752-scrot.png](https://i.postimg.cc/xC8tN3p4/2021-06-25-221739-1364x752-scrot.png)](https://postimg.cc/Q9LpR13Q)
But this colour scheme is not applied to kitten diff
Kitten diff looks like this
[![2021-06-25-221800-1365x749-scrot.png](https://i.postimg.cc/T168VS8P/2021-06-25-221800-1365x749-scrot.png)](https://postimg.cc/tZ2myB40)
Can you please tell me how to change the colour scheme of the kitten diff?

**Comment by kovidgoyal** [comment]
https://sw.kovidgoyal.net/kitty/kittens/diff.html

**Reply by anurag3301** [reply]
Got it, thanks. Just needed to create `diff.conf`. 
I just copy-pasted the config mentioned in #2550. Now Ill cofire to my way.
Is there a way to directly use the kitty's color scheme like which is in this form?
[![2021-06-25-225805-488x470-scrot.png](https://i.postimg.cc/ydmJykpC/2021-06-25-225805-488x470-scrot.png)](https://postimg.cc/7GLYxYyK)

**Comment by benjamin051000** [comment]
Hi, I'm having trouble with this as well. I made a separate discussion here: https://github.com/kovidgoyal/kitty/discussions/5653
